src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationOptions.cs

```diff
@@ -233,6 +234,11 @@ public OpenIdConnectProtocolValidator ProtocolValidator
         public bool RefreshOnIssuerKeyNotFound { get; set; }
 
         /// <summary>
+        /// Determines whether authorization requests should be made using POST.
+        /// </summary>
+        public bool UsePostAuthorizationRequests { get; set; }
```

Comment:
The OIDC middleware currently only uses `Authentication`, though both options are totally valid (OIDC being both an authorization and an authentication protocol). To make this property consistent, I guess it would be better to use `Authentication`.

> An Authentication Request is an OAuth 2.0 Authorization Request that requests that the End-User be authenticated by the Authorization Server.